js/src/main/scala/JSVyxal.scala

```diff
@@ -9,6 +9,10 @@ import scala.scalajs.js.JSConverters.*
 /** A bridge between the interpreter and JS */
 @JSExportTopLevel("Vyxal")
 object JSVyxal:
+
+  var shortDict: Seq[String] = Seq()
+  var longDict: Seq[String] = Seq()
+
   @JSExport
   def execute(
```

Comment:
Another way we could do things is by passing in the short and long dictionaries in the `execute` method itself. No idea what the pros/cons of that are